Tools/setup/ubuntu.sh

```diff
@@ -84,6 +84,7 @@ sudo DEBIAN_FRONTEND=noninteractive apt-get -y --quiet --no-install-recommends i
 	gdb \
 	git \
 	lcov \
+	libfuse2 \ # QGroundControl AppImage requirement
```

Comment:
The comment is giving an error

Reply:
Thank you for the report 🙏  The fix for it is in https://github.com/PX4/PX4-Autopilot/pull/21200 I'd merge it immediately but it's bundled with a compiler upgrade. I think I'll quickly make a separate pr.

Reply:
It was fixed in https://github.com/PX4/PX4-Autopilot/pull/21191 sorry for any inconvenience 🙈